src/main/java/org/broadinstitute/hellbender/tools/exome/plotting/PlotSegmentedCopyRatio.java

```diff
@@ -0,0 +1,242 @@
+package org.broadinstitute.hellbender.tools.exome.plotting;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.*;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.exome.*;
+import org.broadinstitute.hellbender.utils.R.RScriptExecutor;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.io.Resource;
+import org.broadinstitute.hellbender.utils.reference.ReferenceUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@CommandLineProgramProperties(
+        summary = "Create plots of denoised and segmented copy ratio.",
+        oneLineSummary = "Create plots of denoised and segmented copy ratio",
+        programGroup = CopyNumberProgramGroup.class
+)
+public final class PlotSegmentedCopyRatio extends CommandLineProgram {
+    private static final String CNV_PLOTTING_R_LIBRARY = "CNVPlottingLibrary.R";
+    private static final String COPY_RATIO_PLOTTING_R_SCRIPT = "CopyRatioPlotting.R";
+
+    private static final String CONTIG_DELIMITER = ":";
+    private static final int DEFAULT_MINIMUM_CONTIG_LENGTH = 1000000;
```

Comment:
This is mentioned in the parameter documentation, but I can put a comment here as well.